Gemfile.lock

```diff
@@ -249,6 +249,7 @@ GEM
     tzinfo (1.2.5)
       thread_safe (~> 0.1)
     unicode-display_width (1.5.0)
+    wdm (0.1.1)
```

Comment:
please dont change this file

Reply:
Ok
Why did the change happen in my file?

Reply:
because your local environment might have changed these files,
You should avoid overwriting these files because they may cause trouble in deployment.

As a sanity check, we review the PR and look for such changes, so code doesn't break when pushed to prod.